content/python/concepts/built-in-functions/terms/type/type.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'type()'
+Description: 'A function that returns the data type of an object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Built-in Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A function that returns the data type of an object.
+
+## Syntax
+
+```pseudo
+type(some_object)
+```
+
```

Comment:
Can we include a brief sentence or two that explains what all can be used as `some_object` in `type()`?

Reply:
I feel like this is self-explanatory. Is there something specific you believe should be addressed here?